lib/puppet/util/run_mode.rb

```diff
@@ -85,21 +85,21 @@ def code_dir
       end
 
       def var_dir
-        which_dir(File.join(windows_common_base("var")), "~/.puppet/var")
+        which_dir(File.join(windows_common_base("puppet/cache")), "~/.puppet/var")
```

Comment:
If `var` directory sticks around, and we switch `var_dir` to point to `cache`, that could get somewhat misleading/confusing.